app/javascript/group/index.js

```diff
@@ -23,7 +23,7 @@ $(document).ready(async () => {
         </div>`;
       } else if(column === 'platform') {
         barHTML += `
-        <div class="leaderboard-item ${column}-column" style="--ico-url:url(/assets/ico/${group[column].ico}.png);--ico-url-hover:url(/assets/ico/${group[column].ico}_bright.png)"></div>`;
+        <div class="leaderboard-item ${column}-column" style="--ico-url:url(/assets/ico/alces.png);--ico-url-hover:url(/assets/ico/alces_bright.png)"></div>`;
```

Comment:
Left over from testing? I'm getting moosebirds all the way down

Reply:
I'll change my testing approach. It caused too many issues.